### DISCUSSION TITLE
is a disclosure.button a switch?

### DISCUSSION THREAD

**Discussion by jedierikb** [tailwindlabs/headlessui#673] [Help]
Is a disclosure.button a switch?  Could a disclosure.button be a switch?
Seems like shared functionality.  Not sure how to easily integrate these two components.

**Comment by SevenOutman** [comment]
I don't believe it's a `switch`, which is a subclass of `input` by [specification](https://www.w3.org/TR/wai-aria/#switch).

A `button` is a command ("do something") and an `input` is used for data entry in forms.
